src/machine.rs

```diff
@@ -1093,26 +1096,84 @@ impl OlmMachine {
             stream.for_each(move |item| send_user_identities_to_callback(callback_ref, item)).await;
         });
     }
-    /// Register a callback which will be called whenever a secret is received.
+
+    /// Register a callback which will be called whenever a secret
+    /// (`m.secret.send`) is received.
```

Comment:
is it true that it is called for all `m.secret.send` events? I got the impression that some secrets (the cross-signing keys?) were handled internally? I might be wrong though

Reply:
No you are right for now it will only return is the backup decryption key. The cross signing keys are handled internally and custom secret are not yet supported